Source/relay/TourGuide/Sets/Bounty Hunter Hunter.ash

```diff
@@ -55,9 +55,9 @@ ChecklistSubentry SBHHGenerateHunt(string bounty_item_name, int amount_found, in
     
     
     location [int] target_locations;
-    if (amount_needed != -1 && target_monster != $monster[none] && monster_locations.count() > 0)
+    if (amount_needed != 0 && target_monster != $monster[none] && monster_locations.count() > 0)
```

Comment:
structurally all this bounty hunter stuff -looks- fine but candidly i am not sure if it works or not -- my assumption is you have tested it and it all successfully works better than the current set?

Reply:
Yeah, I'm using it for a significant time now, it works great (e.g. when I screech and then sniff, the zone does not disappear in the meantime).